data/templates/get_driveid.js

```diff
@@ -185,6 +192,18 @@ function buildDriveMap(wwidData, vdData, scsiData) {
                 }
             }
         });
+        //vmhbaAdapter:CChannel:TTarget:LLUN
+        //Adapter is set to default value 32 for only one eUsb
+        if (esxiWwid[1] === "mpx.") {
+            esxiWwid[1] = '';
+            if (scsiId !== '') {
+                var hctl = scsiId.split(":");
+                if (hctl.length === 4) {
```

Comment:
[minor]

@pengz1 @lanchongyizu 
in what kind of case will the scsiId  not be 4 digit ?( ex "8:0:0:0") ?
I meant , if that really happen, what consequence will the code turns out to be ?